### DISCUSSION TITLE
kitty on Python Bytes 272

### DISCUSSION THREAD

**Discussion by okken** [kovidgoyal/kitty#4741] [General]
Hi. Just wanted to let you know kitty appeared on the Python Bytes podcast today.
https://youtu.be/8HKliSbA-gQ?t=815

**Comment by kovidgoyal** [comment]
Cool, thanks.

**Comment by kovidgoyal** [comment]
https://github.com/kovidgoyal/kitty/commit/ef4240e19675c6b1345be61213a42403b835b6aa
